### PR TITLE
let .check function return the state, set isVisible on state

### PR DESCRIPTION
**Pull request by kof**

### CODE REVIEW

**Comment by kof** [comment]
not sure why I broke one test

**Comment by joshwnj** [comment]
Taking a look - thanks!

File: visibility-sensor.js [hunk 1]
```diff
@@ -22,6 +22,13 @@ module.exports = React.createClass({
     };
   },
 
+  getInitialState: function () {
+    return {
+      isVisible: true,
```
Comment (joshwnj) [inline visibility-sensor.js]: should `isVisible` be `null` by default? Otherwise the first `onChange` may not be fired.

Comment (kof) [inline visibility-sensor.js]: you are right

Comment (kof) [inline visibility-sensor.js]: fixed

**Comment by kof** [comment]
now its passing

**Comment by joshwnj** [comment]
:+1:

**Pull request merged**